x-pack/plugins/security_solution/common/endpoint/data_generators/base_data_generator.ts

```diff
@@ -9,6 +9,9 @@ import seedrandom from 'seedrandom';
 import uuid from 'uuid';
 
 const OS_FAMILY = ['windows', 'macos', 'linux'];
+const BOOLEANS = [true, false];
```

Comment:
this line by itself.. is something special

Reply:
I know right. Art! 🎨